### PR TITLE
Add Gravatar to the list of Tom's accomplishments

### LINKED ISSUE #1811: What about Tom's association with Gravatar?
I thought @mojombo was associated with Gravatar (used by many, many services).  In this fantastic brag :smile: I don't see any such reference.  Mistake?
 
>Redwood is the latest open source project initiated by Tom Preston-Werner, cofounder of GitHub (most popular code host on the planet), creator of Jekyll (one of the first and most popular static site generators), author of the Semantic Versioning specification (powers the Node packaging ecosystem), and inventor of TOML (an obvious, minimal configuration language used by many projects).

### PR DESCRIPTION
**Pull request by bl-ue**
Closes #1811

/cc @thedavidprice

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ cycles choosing and re-choosing various technologies and configurations. Plus,
 because Redwood is a proper framework, you benefit from continued performance
 and feature upgrades over time and with minimum effort.
 
-Redwood is the latest open source project initiated by Tom Preston-Werner, cofounder of GitHub (most popular code host on the planet), creator of Jekyll (one of the first and most popular static site generators), author of the Semantic Versioning specification (powers the Node packaging ecosystem), and inventor of TOML (an obvious, minimal configuration language used by many projects).
+Redwood is the latest open source project initiated by Tom Preston-Werner, cofounder of GitHub (most popular code host on the planet), creator of Jekyll (one of the first and most popular static site generators), creator of Gravatar (the most popular avatar server on the planet), author of the Semantic Versioning specification (powers the Node packaging ecosystem), and inventor of TOML (an obvious, minimal configuration language used by many projects).
```
Comment (jvanbaarsen) [inline README.md:18]:
```suggestion
Redwood is the latest open source project initiated by Tom Preston-Werner, cofounder of GitHub (most popular code host on the planet), creator of Jekyll (one of the first and most popular static site generators), creator of Gravatar (the most popular avatar service on the planet), author of the Semantic Versioning specification (powers the Node packaging ecosystem), and inventor of TOML (an obvious, minimal configuration language used by many projects).
```

**Review by jvanbaarsen — COMMENTED** [review]
Thanks for your PR :-) I added one suggestion;

**Comment by peterp** [comment]
I don't know why, but this PR heading made me lol.

**Comment by bl-ue** [comment]
Well of course @peterp.  Isn't one enormous cloud product, one configuration language (used by the Rust Foundation), one software development specification, and one templating system used all over the world enough for one person?  Apparently not enough for Tom 😮

**Review by mojombo — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ cycles choosing and re-choosing various technologies and configurations. Plus,
 because Redwood is a proper framework, you benefit from continued performance
 and feature upgrades over time and with minimum effort.
 
-Redwood is the latest open source project initiated by Tom Preston-Werner, cofounder of GitHub (most popular code host on the planet), creator of Jekyll (one of the first and most popular static site generators), author of the Semantic Versioning specification (powers the Node packaging ecosystem), and inventor of TOML (an obvious, minimal configuration language used by many projects).
+Redwood is the latest open source project initiated by Tom Preston-Werner, cofounder of GitHub (most popular code host on the planet), creator of Jekyll (one of the first and most popular static site generators), creator of Gravatar (the most popular avatar service on the planet), author of the Semantic Versioning specification (powers the Node packaging ecosystem), and inventor of TOML (an obvious, minimal configuration language used by many projects).
```
Comment (thedavidprice) [inline README.md:18]:
```suggestion
Redwood is the latest open source project initiated by Tom Preston-Werner, cofounder of GitHub (most popular code host on the planet), creator of Jekyll (one of the first and most popular static site generators), creator of Gravatar (the popular avatar service), author of the Semantic Versioning specification (powers the Node packaging ecosystem), and inventor of TOML (an obvious, minimal configuration language used by many projects).
```

Comment (thedavidprice) [reply README.md:18]: not so sure about the most popular, although instinctively I wouldn't doubt that it is!

File: README.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ cycles choosing and re-choosing various technologies and configurations. Plus,
 because Redwood is a proper framework, you benefit from continued performance
 and feature upgrades over time and with minimum effort.
 
-Redwood is the latest open source project initiated by Tom Preston-Werner, cofounder of GitHub (most popular code host on the planet), creator of Jekyll (one of the first and most popular static site generators), author of the Semantic Versioning specification (powers the Node packaging ecosystem), and inventor of TOML (an obvious, minimal configuration language used by many projects).
+Redwood is the latest open source project initiated by Tom Preston-Werner, cofounder of GitHub (most popular code host on the planet), creator of Jekyll (one of the first and most popular static site generators), creator of Gravatar (the popular avatar service), author of the Semantic Versioning specification (powers the Node packaging ecosystem), and inventor of TOML (an obvious, minimal configuration language used by many projects).
```
Comment (thedavidprice) [inline README.md:18]:
```suggestion
Redwood is the latest open source project initiated by Tom Preston-Werner, cofounder of GitHub (most popular code host on the planet), creator of Jekyll (one of the first and most popular static site generators), creator of Gravatar (the most popular avatar service on the planet), author of the Semantic Versioning specification (powers the Node packaging ecosystem), and inventor of TOML (an obvious, minimal configuration language used by many projects).
```

Ah, I didn't see Tom had approved as is! My misstep. Reverting.

**Comment by thedavidprice** [comment]
Thanks @bl-ue! Very fun indeed.

File: README.md (hunk 2 above)
Comment (bl-ue) [reply README.md:18]: Thought the exact same thing :+1:

**Pull request merged**